client/src/actions/userActions.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+import Types from './types';
+
```

Comment:
This file needed to be made 😁 -- I think the `deleteUser`, `editProfile`, and `addFriend` actions in `authActions` belong here, too.

Reply:
Yeah, we can move those here, but there is only an issue of this file being intended for public user(s) actions, and auth user actions were intended only for the currently logged in user. Couldn't think of a better name, really 😞 Naming things is hard.

Reply:
Ah okay! Maybe those things could go in `userActions` and public profile things could go in `publicProfileActions`?